shared/KOSocket.cpp

```diff
@@ -139,23 +141,27 @@ bool KOSocket::DecryptPacket(uint8 *in_stream, Packet & pkt)
 
 bool KOSocket::Send(Packet * pkt) 
 {
-	if (!IsConnected() || pkt->size() + 1 > GetWriteBuffer().GetAllocatedSize())
+	if (!IsConnected()
+		|| pkt->size() + 7 > MAX_PACKET_SEND_SIZE)
 		return false;
 
-	bool r;
+	WSABUF buf;
+	DWORD w_length = 0;
+	DWORD flags = 0;
+	BYTE out_stream[MAX_PACKET_SEND_SIZE];
+	BYTE Final_Packet[MAX_PACKET_SEND_SIZE];
```

Comment:
```suggestion
	unsigned long w_length = 0;
	uint32_t flags = 0;
	uint8_t out_stream[MAX_PACKET_SEND_SIZE];
	uint8_t Final_Packet[MAX_PACKET_SEND_SIZE];
```

I understand they're used in a WinAPI context, but let's use consistent standard types rather than platform specific.